internal/utilities/util.go

```diff
@@ -1,7 +1,7 @@
 // Copyright 2022 - Offen Authors <hioffen@posteo.de>
 // SPDX-License-Identifier: MPL-2.0
 
-package main
+package utilites
```

Comment:
There's a typo in the package name: `utilities`  has one more `i`.

Reply:
Well now I know, why VS Code insists on adding a package name to the imports :D

Reply:
Ah true, I didn't even think of that.